src/index.html

```diff
@@ -1,16 +1,14 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
-  <title>Authors' Haven</title>
-</head>
-
-<body>
-  <div id="root"></div>
-</body>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Author's Haven</title>
```

Comment:
@nshutijonathan I think you left out this while rebasing `  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
`

Reply:
Yes I have fixed that issue